win32/src/file_win32.c

```diff
@@ -60,4 +41,472 @@ static void on_file_io_complete_win32(
     ULONG io_result,
     ULONG_PTR number_of_bytes_transferred,
     PTP_IO io
-);
\ No newline at end of file
+);
+
+static VOID NTAPI on_close_threadpool_group_member(
+     PVOID object_context, /*what was passed @  CreateThreadpoolIo*/
+     PVOID cleanup_context /*what was passed @  CloseThreadpoolCleanupGroupMembers*/
+)
+{
+    (void)object_context;
+    (void)cleanup_context;
+}
+
+IMPLEMENT_MOCKABLE_FUNCTION(
+    ,

```

Comment:
nit: I'm ok with this style, but maybe we can just leave the first comma at the end of the previous line.

Reply:
done

---
In reply to: [456704266](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456704266) [](ancestors = 456704266)